frankenstein/distributor.go

```diff
@@ -0,0 +1,102 @@
+package frankenstein
+
+import (
+	"sync"
+
+	"github.com/prometheus/prometheus/storage"
```

Comment:
We've always done the opposite order for the last two in scope.  But I'll honor this ordering here.